_posts/2016-03-02-Rust-1.7.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: "Announcing Rust 1.7"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.7. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+As always, you can [install Rust 1.7][install] from the appropriate page on our
+website, and check out the [detailed release notes for 1.7][notes] on GitHub.
+About 1300 patches were landed in this release.
+
+[install]: http://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-170-2016-03-03
+
+### What's in 1.7 stable
+
+This release contains a number of small refinements, one major feature, and
```

Comment:
What is the major feature? :) 

Maybe we should just say something like: "This release is primarily about library stabilization. We stabilized approximately 40 new library functions and methods, including: blah blah blah. 

There are no new language features in this release, though we do have several things cooking for future releases."